app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+class SessionsController < ApplicationController
+  def front
+    redirect_to videos_path if sign_in?
+  end
+
+  def new
+    redirect_to videos_path if sign_in?
+  end
+
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
Dynamic finders such as  `find_by_xxx` is not recommended anymore for single record search. Use `find_by` (without `_email`).

HOMEWORK: figure out why `find_by` is a better option compared to `find_by_email`.